doc/install.text

```diff
@@ -0,0 +1,197 @@
+    - 3 Darks: visit numbers 7291-7293,
+    - 1 Flat: visit number 5,
+    - 1 Arc: visit number 4 
+  Configuration parameters for the pipeline tasks can be set either in config files
+  (see $OBS_PFS_DIR/config/pfs/) or by passing them on the command line (after 
+  "--config", e.g. --config isr.doDark=False). You can list all configuration
+  parameters by appending a "--show config" to the parameter list. Setting "--show
+  config=glob" 
+
+
+
+
+
+
+
+
+
+
+
+
+  First we need to create a directory (actually 2) where we want to store pipeline outputs. 
+  Let's assume you want to store the pipeline outputs in a directory $HOME/spectra/PFS.
+  For the convenience of this quick-start guide we define another environment variable:
+
+export PFS_DATA=$HOME/spectra/PFS
+mkdir -p $PFS_DATA/CALIB
+
+
+* We need to tell the LSST stack which mapper to use. The mapper gives a logical view
+  of both the raw data and the pipeline outputs which you can query for particular data
+  sets. It abstracts out the physical layout so we don't have to worry about the details.
+
+echo "lsst.obs.pfs.PfsMapper" > $PFS_DATA/_mapper
+
+
+* We can now ingest the raw images into a registry stored in $PFS_DATA/registry.sqlite3.
+  The "--mode link" parameter tells the pipeline to create symbolic links instead of copying
+  the raw images. The "-L warn" parameter sets the log level to only print warnings.
+
+ingestImages.py $PFS_DATA $DRP_STELLA_DATA_DIR/tests/data/raw/*.fits --mode link -L warn
+
+
+* Now that we have our database we can start reducing things. We start with creating a master Bias, followed
+  by a Bias-subtracted master Dark. We will then create a Bias- and Dark-subtracted master Flat, which we then use 
+  to identify and trace the apertures of the fiber traces. The fiber traces from the Arc image are then extracted
+  and wavelength calibrated.
+
+  The data we want to reduce were observed/simulated on 2015-12-22 on spectrograph 2, arm r (red) at site S (Summit). 
+
+  The parameter "--rerun USERNAME/rmp" (substitute USERNAME with your name) specifies where to store
+  temporary pipeline outputs.
+  Note the parameter "--cores 1" at the end. This parameter is required by tasks which are parallelized.
+  Sometimes running the code in parallel can lead to problems (in most cases caused by the 3rd-party 
+  libraries used), so setting cores to 1 is a safe choice.
+
+constructBias.py $PFS_DATA --rerun USERNAME/tmp --id field=BIAS dateObs=2015-12-22 arm=r spectrograph=2 --calibId calibVersion=bias calibDate=2015-12-22 arm=r spectrograph=2 --cores 1
+
+
+* Now that we have a master bias we need to ingest that into our calibration database stored in
+  $PFS_DATA/CALIB/calibRegistry.sqlite3. The parameter "--validity 180" specifies that the calibration
+  images are valid for 180 days. We will need to repeat this step every time we create a new calibration
+  image for the butler to find it when its use is required by a later called task.
```

Comment:
(I don't think you should go into a big digression on the Butler here — probably just reword the above to avoid needing to explain it.)